DESCRIPTION

```diff
@@ -0,0 +1,47 @@
+Type: Package
+Package: Covid19UKNowcastsProjections
```

Comment:
I hope I'll never have to type this because there's no way I'd get the capitalisation right ;-)

Reply:
Annoyingly won't allow hyphens so can't match the repo name.